admin-writer/src/main/kotlin/com/epam/drill/admin/writer/rawdata/service/Coverage.kt

```diff
@@ -1,1075 +0,0 @@
-/**
- * Copyright 2020 - 2022 EPAM Systems
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.epam.drill.admin.writer.rawdata.service
-
-import com.epam.drill.admin.writer.rawdata.config.RawDataWriterDatabaseConfig
-import kotlinx.serialization.json.Json
-import kotlinx.serialization.json.JsonObject
-import kotlinx.serialization.json.buildJsonObject
-import kotlinx.serialization.json.encodeToJsonElement
-
-suspend fun versionCoverage(agentId: String, buildVersion: String, rawDataReader: RawDataReader) {
-    val agent = rawDataReader.getAgentConfigs(agentId, buildVersion)
-    val ast = rawDataReader.getAstEntities(agentId, buildVersion)
-    val coverage = rawDataReader.getRawCoverageData(agentId, buildVersion)
-
-    coverage.groupBy { it.className }
-    println("${agent.size} ${ast.size} ${coverage.size}")
-}
-
-suspend fun getNewRisks(newInstanceId: String, oldInstanceId: String): List<JsonObject> {
-    val sqlQuery = risksNewQuery(newInstanceId, oldInstanceId)
-    return executeQuery(sqlQuery)
-}
-
-// TODO suspend is redundant - but it shouldn't be?
-suspend fun executeQuery(sqlQuery: String): List<JsonObject> {
-    val result = mutableListOf<JsonObject>()
-
-    RawDataWriterDatabaseConfig.getDataSource()?.connection.use { connection ->
-        connection?.prepareStatement(sqlQuery)?.use { preparedStatement ->
-            val resultSet = preparedStatement.executeQuery()
-            val metaData = resultSet.metaData
-            val columnCount = metaData.columnCount
-
-            while (resultSet.next()) {
-                val rowObject = buildJsonObject {
-                    for (i in 1..columnCount) {
-                        val columnName = metaData.getColumnName(i)
-                        val columnValue = resultSet.getObject(i)
-                        val stringValue = columnValue?.toString()
-                        put(columnName, Json.encodeToJsonElement(stringValue))
-                    }
-                }
-
-                result.add(rowObject)
-            }
-        }
-    }
-
-    return result
-}
-
-fun generateHtmlTable(results: List<JsonObject>): String {
```

Comment:
I suggest to keep / reimplement something along these lines for our new routes
To provide simple way to get JSON/HTML reports out of our system